Sources/Sham/Models/StubbedDataCollection.swift

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// An object that can be injected into the launch environment to set up the mocking information.
+public final class StubbedDataCollection: Codable {
+    // MARK: Properties
+    
+    /// A dictionary of stubbed responses keyed by stubbed requests.
+    private(set) var stubbedData = [StubRequest: StubResponse]()
+    
+    // MARK: Functions
+    
+    public init() {}
+    
+    // MARK: Stubbing
+    
+    /// Adds a response to the stub response collection for the MockService.
```

Comment:
The last bit can be removed since this class has no knowledge of `MockService.swift`
```suggestion
    /// Adds a response to the stub response collection.
```